packages/semantic-tokens/utils/fluentOverrides.test.ts

```diff
@@ -5,6 +5,7 @@ describe('Ensure all fluentOverrides are valid tokens', () => {
   // We'll use this to catch any breaking changes in tokens.json
   it('Splits and camel cases strings separated by forward slash', () => {
     Object.keys(fluentOverrides).forEach(fluentOverrideKey => {
+      console.log(`Checking ${fluentOverrideKey}`);
```

Comment:
This is useful to make sure the test is actually debuggable, but not sure if we should keep